src/dune_tui/dune_tui.ml

```diff
@@ -36,7 +36,9 @@ module Tui = struct
     fun () -> !(Lazy.force setup)
   ;;
 
-  let start () = Unix.set_nonblock Unix.stdin
+  let term_input_fds, _ = Term.fds (term ())
+
+  let start () = Unix.set_nonblock term_input_fds
```

Comment:
That's wrong. You can't do that anymore

Reply:
That snuck in during rebase, shouldn't be there.